src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.CompletionListUpdater.cs

```diff
@@ -577,7 +577,7 @@ private ImmutableArray<CompletionFilterWithState> GetUpdatedFilters(IReadOnlyLis
             }
 
             /// <summary>
-            /// Given multiple possible chosen completion items, pick the one using the following perferences (in order):
+            /// Given multiple possible chosen completion items, pick the one using the following preferences (in order):
```

Comment:
somewhere in my heart I hope this is due to the spellchecking engine

Reply:
Yes indeed :)